wbooks/src/app/App.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import logoWolox from './assets/LogoWolox.png';
 import './application.css';
+import PropTypes from 'prop-types';
```

Comment:
You can extract directly here the props:

import { string } from 'prop-types'

Reply:
done!